src/xrpld/ledger/detail/View.cpp

```diff
@@ -169,6 +169,18 @@ hasExpired(ReadView const& view, std::optional<std::uint32_t> const& exp)
     return exp && (view.parentCloseTime() >= tp{d{*exp}});
 }
 
+bool
+noDefaultRipple(ReadView const& view, Issue const& issue)
```

Comment:
Could you please add optional issuer account argument, cause it is likely it will be available at the time this function needed, and we can save drop `view.read` call

Reply:
I only have the sleIssuer in EscrowCreate so I will just do the manual check there and leave this function as is.